### ISSUE TITLE
Some questions about using sparse depth maps directly as FusionNet inputs

### ISSUE THREAD

**Issue by Cajeput** [nesl/radar-camera-fusion-depth#22]
In the code you provided, FusionNet training requires depth and response to obtain input depth, but when sparse depth maps are directly input into FusionNet (i.e. no RadarNet), how can this response be obtained?
[input_depth = torch.cat([depth, response], dim=1)
 output_depth = fusionnet_model.forward(image=image, input_depth=input_depth)](url)

**Comment by Cajeput** [comment]
Now that the sparse depth map obtained by projecting the original radar points directly onto the image plane has been obtained, how can it be input to Fusionnet (in the case of no RadarNet)? How do I modify the sh script to perform FusionNet training with no RadarNet?

**Comment by Cajeput** [comment]
Hello, can you help me answer the above question? @thecyclone

**Comment by Cajeput** [comment]
I still hope you can answer my question:
My question is that you mentioned in your paper "simply training FusionNet using raw radar points", but the inputs to FusionNet include semi dense radar depth maps and corresponding confidence maps. Can you explain how to train FusionNet using raw radar points?

**Comment by zhuzhaohuan** [comment]
Hello, I noticed in your previous issue that you downloaded and utilized a pre-trained model. I was wondering if you experienced any issues with the pre-trained model in the scripts 'run_fusionnet_nuscenes_test.sh' and 'setup_dataset_nuscenes_radarnet_test.sh', as depicted in the figure below?
![image](https://github.com/nesl/radar-camera-fusion-depth/assets/48746939/be31a613-63bb-4d9e-8ca7-29a50155e633)

**Comment by zhuzhaohuan** [comment]
> I still hope you can answer my question: My question is that you mentioned in your paper "simply training FusionNet using raw radar points", but the inputs to FusionNet include semi dense radar depth maps and corresponding confidence maps. Can you explain how to train FusionNet using raw radar points?

Hello, I noticed in your previous issue that you downloaded and utilized a pre-trained model. I was wondering if you experienced any issues with the pre-trained model in the scripts 'run_fusionnet_nuscenes_test.sh' and 'setup_dataset_nuscenes_radarnet_test.sh', as depicted in the figure below?
<img width="742" alt="8af78d5a9d2e8b06cf02387f9107118" src="https://github.com/nesl/radar-camera-fusion-depth/assets/48746939/695fc203-e930-47ba-acd6-763bf88ba916">

**Comment by thecyclone** [comment]
Unfortunately, I do not have access to a GPU machine so I cannot give you the code or test anything for you. You will need to  implement it yourself. Thanks
